src/test/java/htsjdk/samtools/CRAMIteratorTest.java

```diff
@@ -0,0 +1,50 @@
+package htsjdk.samtools;
+
+import htsjdk.HtsjdkTest;
+import htsjdk.samtools.cram.ref.ReferenceSource;
+import htsjdk.samtools.seekablestream.SeekableStream;
+import org.junit.Assert;
+import org.testng.annotations.Test;
+
+import java.io.File;
+
+
+/**
+ * @author Anton_Mazur@epam.com, EPAM Systems, Inc.
```

Comment:
This is a test file, so it doesn't matter, but block comments like "author" need to go after the main javadoc or else the entire javadoc text becomes part of the author block.